waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -75,9 +75,9 @@
 ;; Fails on (is (> (count instance-ids) 1) (str instance-ids)) as there is only one instance
 (deftest ^:parallel ^:integration-slow ^:resource-heavy test-s3-logs
   (testing-using-waiter-url
-    (when (using-k8s? waiter-url)
-      (let [log-bucket-url (-> waiter-url get-kubernetes-scheduler-settings :log-bucket-url)
```

Comment:
I think changing this `let` to a `when-let` makes the test simpler to read.

Reply:
Refactored